fenix/app/src/main/java/org/mozilla/fenix/home/recentvisits/view/RecentlyVisited.kt

```diff
@@ -231,32 +233,34 @@ private fun RecentlyVisitedHistoryGroup(
 
         Spacer(modifier = Modifier.width(imageSpacer))
 
-        Column(
-            modifier = Modifier.fillMaxSize(),
-        ) {
-            RecentlyVisitedTitle(
-                text = recentVisit.title,
-                modifier = Modifier
-                    .padding(top = 7.dp, bottom = 2.dp)
-                    .weight(1f)
-                    .semantics {
-                        testTagsAsResourceId = true
-                        testTag = "recent.visits.group.title"
-                    },
-            )
+        Box(modifier = Modifier.fillMaxSize()) {
```

Comment:
follow-up for future homescreen work: this could be a good candidate to be replaced by our [`IconListItem` top level composable](https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/java/org/mozilla/fenix/compose/list/ListItem.kt#98).

Reply:
created a follow up👍